app/src/main/res/values-hi/strings.xml

```diff
@@ -110,6 +110,7 @@
     <string name="empty_repayment_schedule">आपके साथ संबद्ध कोई भी पुनर्भुगतान शेड्यूल नहीं है</string>
     <string name="no_more_transactions_available">कोई और लेनदेन उपलब्ध नहीं है</string>
     <string name="no_transaction">कोई लेनदेन नहीं</string>
+    <string name="transfer_error">स्थानांतरण पूर्ण करने में असमर्थ, कृपया बाद में फिर से प्रयास करेंं</string>
```

Comment:
What language is this, just curious? Just remember that unless this is your native language, by GCI rules, you are not allow to add translations. :)

Reply:
@mdew192837 This language is Hindi, and it is needed to be added as Mifos Mobile App gives two language options Hindi and English as of now

Reply:
@mdew192837 we are supporting Hindi too.